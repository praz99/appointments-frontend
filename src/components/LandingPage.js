import React from 'react';
import { Link } from 'react-router-dom';
import Login from '../layouts/Login';
import '../styles/Landing.css';

const LandingPage = () => (
  <div className="landing-container">
    <div className="landing-info">
      <h1>Welcome to The Homes</h1>
      <p>Explore through the variety of houses.</p>
      <p>Find your dream house..</p>
      <p>Book an appointment and visit the house anytime you want...</p>
    </div>
    <div className="landing-login">
      <h3>Lets get started</h3>
      <Login />
      <div className="landing-signup-info">
        <p>Don&#39;t have an account?</p>
        <Link to="/signup">Signup</Link>
      </div>
    </div>
  </div>
);

export default LandingPage;
