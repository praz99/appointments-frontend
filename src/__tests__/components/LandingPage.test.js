import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import LandingPage from '../../components/LandingPage';

configure({ adapter: new Adapter() });

it('renders LandingPage component correctly', () => {
  const landing = shallow(<LandingPage />);
  expect(landing).toMatchSnapshot();
});
