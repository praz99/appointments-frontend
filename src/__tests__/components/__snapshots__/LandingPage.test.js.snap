// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders LandingPage component correctly 1`] = `ShallowWrapper {}`;
